public/editor/scripts/editor/js/fc/bramble-ui-bridge.js

```diff
@@ -13,6 +13,7 @@ define(function(require) {
 
   var adapting = false;
   var adaptTimeoutMS = 200; // How often we adapt editor bar layout
+  var _isMobileView = false; // to check wheather mobile view is on/off
```

Comment:
please don't use underscores in variables: there is nothing special about this variable and it deserves a normal name. Also on a typo issue, `wheather` -> `whether`